tox.ini

```diff
@@ -22,3 +22,10 @@ deps =
     black==18.9b0
 commands =
     black {posargs:--check setup.py faculty_cli test}
+
+# This section is in use Python 2 is supported
+# With Python 3 flake8-black would unify the requirements
+# of the formatter and linter.
+[flake8]
+# Longer line length is due to black's default line wrap
+max-line-length = 88
```

Comment:
This shouldn't be necessary: we configure Black to use a maximum line length of 79 [in `pyproject.toml`](https://github.com/facultyai/faculty-cli/blob/96b83abc2fbe639e2747dd17e614edc2133d0160/pyproject.toml#L2).

Reply:
Unfortunately, this is necessary, and not even really sufficient, as flake8 and black checks things according to different settings. See for example this CI run where formatting is as black's done it, but not acceptable for flake8 https://travis-ci.org/github/facultyai/faculty-cli/builds/664454727 IMHO ideally the same set of rules should be used for both (e.g. with `flake8-black`, but as noted, that library is not available for Python 2.

Looking at the circumstances, I'll drop this part as indeed inefective, and trying to treat symptomes (though IMHO the same way as pyproject.toml is trying to treat the symptoms of the two tools expecting different standards). But the issue of requiring manual reformatting still remains, as `black` does not wrap long strings https://github.com/psf/black/issues/182
I've checked with `black --verbose`, and `pyproject.toml` is indeed taken account overall, but that still allows long lines coming from strings where.

Thus if e.g. this is run through `black`, it will be satisfied as is
```
somestring = "123456789 123456789 123456789 123456789 123456789 123456789 123456789 123456789 123456789 "
```
```
test.py wasn't modified on disk since last run.
All done! ✨ 🍰 ✨
1 file left unchanged.
```
but `flake8` will not let it pass:
```
test.py:1:80: E501 line too long (104 > 79 characters)
```

As a sidenote, I am not quite sure of the value of running flake8 both with Python 2 and 3, so would be interested to know. If only Python 3 is used then these rules can be reconciled easier.

Reply:
Ah, I see. As we're enforcing Black for formatting I'd be content to just disable flake8's E501 error.

Reply:
Yeah, that's a good approach as well, almost anything is good than developers spending time trying to creatively work around artificial limitations.